packages/eslint-config/index.js

```diff
@@ -121,22 +119,15 @@ module.exports = [
         },
       ],
       "tsdoc/syntax": "warn",
-      "imports/imports": "warn",
-      "imports/exports": "warn",
-    },
-  },
-  // ts declarations
-  {
-    files: ["**/*.d.ts"],
```

Comment:
This was only required because I was missing the `declare` keywords in manually written `.d.ts` files.